pkg/cmd/roachtest/tests/typeorm.go

```diff
@@ -24,7 +24,7 @@ import (
 )
 
 var typeORMReleaseTagRegex = regexp.MustCompile(`^(?P<major>\d+)\.(?P<minor>\d+)\.(?P<point>\d+)$`)
-var supportedTypeORMRelease = "0.2.32"
+var supportedTypeORMRelease = "0.3.4"
```

Comment:
holding off until 0.3.5 is released since we want https://github.com/typeorm/typeorm/commit/ec6c11920b70132396cd2e098b378b753f6e4d3f